agent/test/unit/com/thoughtworks/go/agent/AgentWebSocketClientControllerTest.java

```diff
@@ -106,6 +102,8 @@
     @After
     public void tearDown() {
         GuidService.deleteGuid();
+        System.clearProperty("go.agent.websocket.enabled");
```

Comment:
Can you change this to something like:

```
System.clearProperty(SystemEnvironment.WEBSOCKET_ENABLED.propertyName());        
System.clearProperty(SystemEnvironment.CONSOLE_LOGS_THROUGH_WEBSOCKET_ENABLED.propertyName());
```

so that the keys are not duplicated?